app/api/tax.py

```diff
@@ -43,9 +65,38 @@ class Meta:
                          type_='event')
 
 
-class TaxList(ResourceList):
+class TaxListPost(ResourceList):
     def before_post(self, args, kwargs, data):
         require_relationship(['event'], data)
+        if not has_access('is_coorganizer', event_id=data['event']):
+            raise ForbiddenException({'source': ''}, 'Co-organizer access is required.')
+
+    def before_get(self, args, kwargs):
+        if has_access('is_admin'):
```

Comment:
So if this TaxSchemaPublic list is visible by public then it should be "is_admin_with_jwt" also because it will throw 403 for any guest